int64 id
float64 p_happy
float64 p_sad
float64 p_neutral

#message file containing 1 int64 discrete value and 3 float values
